keras_cv/models/vgg16.py

```diff
@@ -25,35 +25,62 @@
 from keras_cv.models import utils
 
 
-def VGG16(
-    *,
-    include_rescaling,
-    include_top,
-    classes=None,
-    weights=None,
-    input_shape=(224, 224, 3),
-    input_tensor=None,
-    pooling=None,
-    classifier_activation="softmax",
-    name="VGG16",
-    **kwargs,
+def apply_vgg_block(
+    x,
+    num_layers,
+    kernel_size,
+    stride,
+    activation,
+    padding,
+    max_pool,
+    name,
 ):
+    """
+    Builds VGG block
+    Args:
+        x: None (or) Tensor, input tensor to pass through network
```

Comment:
nit: I don't think x can be `None` here -- only a Tensor

Reply:
Done.